source/navitiacommon/navitiacommon/default_values.py

```diff
@@ -80,6 +80,17 @@
 #only journey with this mode (or slower mode) will be used for calculate the max_duration
 max_duration_fallback_mode = 'walking'
 
+# specify the latest time point of request, relative to datetime, in second
+max_duration = 60*60*24  # seconds
+
+# the penalty for each walking transfer
+walking_transfer_penalty = 2  # minutes
```

Comment:
All other duration are in seconds, there is a specific reason to use minutes for this one?